lib/commit_after_review.js

```diff
@@ -0,0 +1,76 @@
+function commitAfterReview(commit, review, logger) {
+  let commitIndex = commit.length === 0
+    ? 0 : commit.length - 1;
+  let reviewIndex = review.length === 0
+    ? 0 : review.length - 1;
+
+  let lastCommit = commit[commitIndex].commit;
+  let lastReview = review[reviewIndex];
+
+  let commitInfo = getDate(lastCommit.committedDate);
+  let reviewInfo = getDate(lastReview.publishedAt);
+
+  logger.info('checking if new commits were pushed' +
```

Comment:
Is this necessary? There's no async processing here or anything...

Reply:
I will remove that